wgsl/index.bs

```diff
@@ -1134,15 +1134,44 @@ postfix_expression
  
 argument_expression_list
   : (logical_or_expression COMMA)* logical_or_expression
- 
+
+singular_expression
+  : unary_expression
+  | intrinsic_expression
+  | derivative_function (LESS_THAN derivative_modifier GREATER_THAN)? PAREN_LEFT IDENT PAREN_RIGHT
+
 unary_expression
   : primary_expression postfix_expression
   | MINUS unary_expression
       OpSNegate
       OpFNegate
   | BANG unary_expression
```

Comment:
This means you can't do `!IsAny()` anymore.

Reply:
yeah, I'll need to refactor this bit

Reply:
@dj2 how about now?